src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ObjectExplorerService.cs

```diff
@@ -365,7 +359,7 @@ private async Task<SessionCreatedParameters> CreateSessionAsync(ConnectionDetail
                 response = new SessionCreatedParameters
                 {
                     Success = true,
-                    RootNode = session.Root.ToNodeInfo(),
+                    RootNode = new NodeInfo(session.Root),

```

Comment:
Out of curiosity, was there a specific reason you removed the helper method? Seems useful to have still.

Reply:
I wanted to separate out contract classes from core OE classes. Therefore, I removed this helper node from TreeNode.

Reply:
You could also have an extension method instead, but it's fine if you wanted to leave it this way. I just like to have stuff like that be discoverable through autocomplete instead of having to just know that this NodeInfo type exists out there and has a constructor that takes in the node.